spec/controllers/api/v1/work_groups_controller_spec.rb

```diff
@@ -204,4 +205,75 @@
       end
     end
   end
+
+  describe '#initiate' do
+    context 'success' do
+      it 'sets the progess status of a work group to in_progress' do
+        patch :initiate, params: { id: work_group2.id, format: :json }
+        expect(controller.instance_variable_get(:@work_group).aasm_state).to eq('in_progress')
+        expect(response).to have_http_status(200)
+      end
+    end
+
+    context 'failure' do
```

Comment:
check if the body returns an error message with more info and if yes add the check to the test